File: lang-java/src/test/resources/fixtures/nested-lookup/positive/list-contains-in-for.java
import java.util.List;

public class ListContainsInFor {
    public void findMatches(List<String> items, List<String> targets) {
        for (String item : items) {
            if (targets.contains(item)) {
                System.out.println(item);
            }
        }
    }
}

File: lang-kotlin/src/test/resources/fixtures/nested-lookup/positive/list-contains-in-for.kt
class OrderFilter {
    fun findMatches(items: List<String>, targets: List<String>): List<String> {
        val result = mutableListOf<String>()
        for (item in items) {
            if (targets.contains(item)) {
                result.add(item)
            }
        }
        return result
    }
}
